src/components/calcite-button/calcite-button.tsx

```diff
@@ -252,4 +278,15 @@ export class CalciteButton {
       e.preventDefault();
     }
   };
+
+  private renderLoader(): VNode {
+    if (this.loading) {
```

Comment:
Can this be done with JSX instead of manually working with the DOM? I think you could use a `State` prop to help control the animation classes.